modules/mono/glue/GodotSharp/.editorconfig

```diff
@@ -10,3 +10,8 @@ dotnet_diagnostic.CA1708.severity = none
 dotnet_diagnostic.CS1591.severity = none
 # CS1573: Parameter has no matching param tag in the XML comment
 dotnet_diagnostic.CS1573.severity = none
+
+[GodotSharp/Core/**.cs]
+# CS1591: Missing XML comment for publicly visible type or member
+# Temporary change to not pollute the warnings, but we need to document public APIs
```

Comment:
nit: Since it's temporary it may be easier to keep track of it with a `TODO:`.

```suggestion
# TODO: Temporary change to not pollute the warnings, but we need to document public APIs
```

Reply:
Good point, added!